src/lib/bootloader/bootloader_base.rb

```diff
@@ -61,9 +62,9 @@ def packages
     end
 
     # done in common write but also in installation pre write as kernel update need it
```

Comment:
`pre_write` is self-explanatory, but maybe adding the usual `@param` documentation would be nice.